test/loginjection.test.ts

```diff
@@ -54,32 +63,83 @@ describe('log injection', () => {
 
   describe('default flow', () => {
```

Comment:
Is there a test for disabled log injection?

Reply:
This PR removes the option to toggle it. There's already a test for testing overwriting the default.